goodtables/compat.py

```diff
@@ -45,22 +46,28 @@ def iterenc_utf8(data):
 
         iter = chain(first_lines, data)
         iter = iterenc_utf8(iter)
-        reader = csv.reader(iter, dialect=dialect, **kwargs)
-        for row in reader:
-            yield [str(cell, 'utf-8') for cell in row]
+        csv.field_size_limit(20000000)
```

Comment:
@georgiana-b you cant add a file size limit here. We need to be able to accept huge files, so this is not a solution.